Projects/MoveFlow/Sources/View/MovingFlowHousingType.swift

```diff
@@ -3,54 +3,74 @@ import SwiftUI
 import hCore
 import hCoreUI
 
-public struct MovingFlowHousingType: View {
-    @PresentableStore var store: MoveFlowStore
-    let housingTypes = ["Bostadsrätt", "Hyresrätt", "Villa"]
-    @State var isSelected: String = ""
-
-    public init() {
-        store.send(.getMoveIntent)
-    }
+public struct MovingFlowHousingTypeView: View {
+    @ObservedObject var vm = MovingFlowHousingTypeViewModel()
 
     public var body: some View {
-
         LoadingViewWithContent(MoveFlowStore.self, [.fetchMoveIntent], [.getMoveIntent]) {
-            hForm {
-                VStack {
-                    ForEach(housingTypes, id: \.self) { type in
-                        setCheckBoxComponent(text: type)
+            hForm {}
+                .hFormTitle(.standard, .title3, L10n.changeAddressSelectHousingTypeTitle)
```

Comment:
The text font doesn't look the same as in the figma design. Should be standard like it is on the other places